### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Big-Hdub**
All changes worked on 7-10-24,  seeders added for production

### CODE REVIEW

File: react-vite/src/redux/questions.js [hunk 1]
```diff
@@ -12,7 +13,7 @@ const setCurrentQuestion = (question) => ({
 })
 
 export const thunkGetAllQuestions = () => async (dispatch) => {
-    const response = await fetch('/api/questions/');
+    const response = await fetch('/api/questions');
```
Comment (Forestbriggs) [inline react-vite/src/redux/questions.js:16]: Add the forward slashes back to keep gunicorn from rerouting to http

File: react-vite/src/components/TestAnswer/TestAnswer.jsx
Comment (Forestbriggs) [inline react-vite/src/components/TestAnswer/TestAnswer.jsx:1]: Why are you pushing a test route to dev or prod?

File: app/api/question_routes.py [hunk 2]
```diff
@@ -41,33 +41,38 @@ def user_saves():
...
+        returns all details of a question
+    """
     question = Question.query.get(question_id)
-    
+
     if not question:
         return {'errors': {'message': 'Question could not be found'}}, 404
-    
+
+    print('question details:', question.to_dict_details())
```
Comment (Forestbriggs) [inline app/api/question_routes.py]: Need to get rid of print statement

File: app/api/question_routes.py [hunk 3]
```diff
@@ -41,33 +41,38 @@ def user_saves():
...
-    
+
     form = QuestionForm()
+    form['csrf_token'].data = request.cookies['csrf_token']
     if form.validate_on_submit():
         new_question = Question(
             user_id = current_user.id,
             title = form.title.data,
             details = form.details.data,
             expectation = form.expectation.data
```
Comment (Forestbriggs) [inline app/api/question_routes.py:71]: Need to add created_at and updated_at with datetime.now() to post route

File: app/api/question_routes.py [hunk 4]
```diff
@@ -41,33 +41,38 @@ def user_saves():
...
         )
-        
+
         db.session.add(new_question)
         db.session.commit()
         return new_question.to_dict(), 201
```
Comment (Forestbriggs) [inline app/api/question_routes.py:76]: Need to return form errors unindented after this return statement for when form does not validate

File: app/forms/create_comment_form.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+from flask_wtf import FlaskForm
+from wtforms.fields import TextAreaField
+from wtforms.validators import DataRequired, Length
+
+class CommentForm(FlaskForm):
+    comment = TextAreaField('Comment', validators=[DataRequired(), Length(max=255)])
```
Comment (Forestbriggs) [inline app/forms/create_comment_form.py]: Comment length should be 1500

File: app/forms/create_question_form.py [hunk 6]
```diff
@@ -5,4 +5,4 @@
 class QuestionForm(FlaskForm):
     title = StringField('Title', validators=[DataRequired(), Length(max=50)])
```
Comment (Forestbriggs) [inline app/forms/create_question_form.py]: title length should be 150

File: app/seeds/questions.py [hunk 7]
```diff
@@ -192,11 +192,11 @@ def random_datetime(start_date, end_date):
 def seed_questions():
     for info in user_questions:
         username, questions = info.values()
-        print('username', username)
-        print('questions', questions)
+        # print('username', username)
```
Comment (Forestbriggs) [inline app/seeds/questions.py:195]: delete prints

File: react-vite/dist/index.html
Comment (Forestbriggs) [inline react-vite/dist/index.html:1]: rerun build after front end changes

File: react-vite/src/components/TestAnswer/TestAnswer.jsx
Comment (Forestbriggs) [inline react-vite/src/components/TestAnswer/TestAnswer.jsx:1]: Why are you pushing  a test route to dev or prod? Please remove from dev.

File: react-vite/src/router/index.jsx [hunk 8]
```diff
@@ -36,6 +37,10 @@ export const router = createBrowserRouter([
                 path: "signup",
                 element: <SignupFormPage />,
             },
+            {
+                path: "test",
```
Comment (Forestbriggs) [inline react-vite/src/router/index.jsx]: get rid of test route in router pls

**Review by Forestbriggs — DISMISSED** [review]

File: app/api/question_routes.py (hunk 3 above)
Comment (Big-Hdub) [reply app/api/question_routes.py:71]: That is the default that is done by the migration and model

File: app/seeds/questions.py (hunk 7 above)
Comment (Big-Hdub) [reply app/seeds/questions.py:195]: they are deleted with one that is commented out.

File: app/api/question_routes.py (hunk 2 above)
Comment (Big-Hdub) [reply app/api/question_routes.py]: done

**Review by Sohna-AI — APPROVED** [review]
Made the requested changes

**Pull request merged**